### PR TITLE
MET-5564 Add Documentation Arrows

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: projects/sandbox/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -31,6 +31,32 @@ const routes: Routes = [
       reuseComponent: true
     }
   },
+  {
+    path: 'x',
```
Comment (jeortizquan) [inline projects/sandbox/src/app/app-routing.module.ts:35]: sandbox easter egg :joy: 

https://elgoog.im/matrix/3d/

Comment (jeortizquan) [reply projects/sandbox/src/app/app-routing.module.ts:35]: afbeelding

File: projects/sandbox/src/app/doc-arrows/doc-arrows.component.ts [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+  arrowActiveKey(event: KeyboardEvent): void {
+    const tgt = event.target as HTMLElement;
+
+    if (tgt) {
+      const arrow = tgt.closest('.arrow') as HTMLElement;
+
+      if (arrow) {
+        // rotation
+        if (['r', 'R'].includes(event.key)) {
```
Comment (jeortizquan) [inline projects/sandbox/src/app/doc-arrows/doc-arrows.component.ts:97]: only writing `r` works caps `R` at the moment is not possible to use.

afbeelding

**Review by jeortizquan — COMMENTED** [review]
My suggestion would be to add a small tooltip, reminding user controls
```
use Arrows to change size.
SHIFT+R to rotate arrow.
CTRL+Arrow to move bigger step.
Backspace to delete arrow.
```

afbeelding

File: projects/sandbox/src/app/doc-arrows/doc-arrows.component.ts (hunk 2 above)
Comment (andyjmaclean) [reply projects/sandbox/src/app/doc-arrows/doc-arrows.component.ts:97]: If there's a diagram using letters up to 'R' then it probably has too many arrows!

But yes, they'd have to use cap lock in this case.

**Pull request merged**